sdk/core/azure-core-serializer-avro-jackson/src/main/java/com/azure/core/serializer/avro/jackson/JacksonAvroSerializer.java

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.serializer.avro.jackson;
+
+import com.azure.core.serializer.AvroSerializer;
+import com.fasterxml.jackson.dataformat.avro.AvroMapper;
+import com.fasterxml.jackson.dataformat.avro.AvroSchema;
+import reactor.core.Exceptions;
+import reactor.core.publisher.Mono;
+
+import java.io.IOException;
+import java.io.OutputStream;
+
+/**
+ * Jackson based implementation of the {@link AvroSerializer} interface.
+ */
+public final class JacksonAvroSerializer implements AvroSerializer<AvroSchema> {
+    private final AvroMapper mapper;
+
+    /**
+     * Constructs a {@link AvroSerializer} using the default Jackson serializer.
+     */
+    public JacksonAvroSerializer() {
+        this.mapper = new AvroMapper();
+    }
+
+    /**
+     * Constructs a {@link AvroSerializer} using the passed Jackson serializer.
+     *
+     * @param mapper Configured Jackson serializer.
+     */
+    public JacksonAvroSerializer(AvroMapper mapper) {
+        this.mapper = mapper.copy();
```

Comment:
Can this just used the provided mapper instead of making a copy?

Reply:
Removed the `copy` call.